frontend/src/scenes/session-recordings/player/share/playerShareLogic.ts

```diff
@@ -17,34 +36,76 @@ export const playerShareLogic = kea<playerShareLogicType>([
+                        ? 'Set a valid time like 02:30 (minutes:seconds)'
+                        : undefined,
+            }),
+            options: {
+                // whether we show errors after touch (true) or submit (false)
+                showErrorsOnTouch: true,
+
+                // show errors even without submitting first
+                alwaysShowErrors: true,
+            },
+        },
+        linearLinkForm: {
+            defaults: {
+                includeTime: true,
+                time: colonDelimitedDuration(props.seconds, null),
+                issueTitle: '',
+                issueDescription: '',
+            } as FormWithTime & {
+                issueTitle: string
+                issueDescription: string
             },
             errors: ({ time, includeTime }) => ({
                 time:
                     time && includeTime && reverseColonDelimitedDuration(time || undefined) === null
                         ? 'Set a valid time like 02:30 (minutes:seconds)'
                         : undefined,
             }),
+            options: {
+                // whether we show errors after touch (true) or submit (false)
+                showErrorsOnTouch: true,
+
+                // show errors even without submitting first
+                alwaysShowErrors: true,
+            },
         },
     })),
 
     selectors(({ props }) => ({
-        queryParams: [
-            (s) => [s.shareUrl, s.shareUrlHasErrors],
-            (shareUrl, hasErrors) => {
-                if (!shareUrl.includeTime || hasErrors) {
-                    return {}
+        privateLinkUrlQueryParams: [
+            (s) => [s.privateLinkForm],
+            (privateLinkForm) => {
+                return makePrivateLinkQueryParams(privateLinkForm)
+            },
+        ],
+        privateLinkUrl: [
+            (s) => [s.privateLinkForm],
+            (privateLinkForm) => {
+                return makePrivateLink(props.id, privateLinkForm)
+            },
+        ],
+        linearQueryParams: [
+            (s) => [s.linearLinkForm],
+            (linearLinkForm) => {
+                return {
+                    title: linearLinkForm.issueTitle,
+                    description:
+                        linearLinkForm.issueDescription +
+                        `\n\nPostHog recording: ${makePrivateLink(props.id, linearLinkForm)}`,
                 }
-                return { t: `${reverseColonDelimitedDuration(shareUrl.time) || 0}` }
             },
         ],
-        url: [
-            (s) => [s.queryParams],
-            (queryParams) => {
-                return combineUrl(urls.absolute(urls.currentProject(urls.replaySingle(props.id))), queryParams).url
+        linearUrl: [
+            (s) => [s.linearQueryParams],
+            (linearQueryParams) => {
+                return combineUrl('https://linear.app/new', linearQueryParams).url
```

Comment:
https://developers.linear.app/docs/guides/how-to-create-new-issues-using-linear.new